src/index.html

```diff
@@ -7,23 +7,25 @@
   <title>Leaderboard</title>
 </head>
 <body>
-  <main class="p-3">
-    <h1>LeaderBoard</h1>
-    <div class="d-flex justify-content-around">
-      <div class="d-flex flex-column">
-        <div class="d-flex justify-content-between">
-          <h2>Recent Scores</h2>
-          <button type="button">Refresh</button>
+  <main class="p-3 mx-auto">
+    <h1 class="pb-5">LeaderBoard</h1>
+    <div class="d-flex justify-content-center flex-wrap">
+      <div class="left-content d-flex flex-column">
+        <div class="left-content-header d-flex">
+          <h2 class="me-4">Recent Scores</h2>
+          <button id="refresh-score" class="default-button" type="button">Refresh</button>
         </div>
-        <ul id="leaderboard-container" class="list-unstyled border border-2 border-dark">
+        <ul id="leaderboard-container" class="list-unstyled">
         </ul>
       </div>
 
-      <div class="d-flex flex-column">
-        <h2>Add your Score</h2>
-        <input type="text" placeholder="Your name">
-        <input type="text" placeholder="Your score">
-        <button class="ms-auto" type="button">Submit</button>
+      <div class="right-content d-flex flex-column">
+        <div class="right-content-header">
+          <h2>Add your Score</h2>
+        </div>
+        <input id="input-user" type="text" placeholder="Your name">
+        <input id="input-score" type="text" placeholder="Your score">
+        <button id="add-score" class="ms-auto" type="button">Submit</button>
```

Comment:
- [ ] Score input type should be a number not a text.
![arr32](https://user-images.githubusercontent.com/55635977/132052988-b96a115f-3948-45d4-b6d3-6c20d359d238.png)